apps/search/src/domain/WebhookActivityToggler.service.ts

```diff
@@ -0,0 +1,119 @@
+import { Client, OperationResult } from "urql";
+import {
+  DisableWebhookDocument,
+  EnableWebhookDocument,
+  FetchOwnWebhooksDocument,
+} from "../../generated/graphql";
+import { createLogger } from "../lib/logger";
+
+const logger = createLogger({
+  service: "WebhookActivityTogglerService",
+});
+
+export interface IWebhooksClient {
+  fetchAppWebhooksIDs(id: string): Promise<string[]>;
+  disableSingleWebhook(id: string): Promise<void>;
+  enableSingleWebhook(id: string): Promise<void>;
+}
+
+export class WebhooksClient implements IWebhooksClient {
```

Comment:
Nitpick: `WebhooksClient` is quite a generic name for an exported class. Given the name of the class below, I would name it something like `WebhooksActivityClient`.

Reply:
improved the name